src/scripts/app.js

```diff
@@ -1 +1,22 @@
 console.log('hello');
+
+Array.from(document.querySelectorAll('.stars')).forEach(function(a) {
+  a.addEventListener('click', function(e) {
+    e.preventDefault();
+    console.log(e);
+    e.originalTarget.classList.add('full');
```

Comment:
Masz aktualnego mastera? We wcześniejszym PR też pisałeś `e.originalTarget` i poprawiałeś to - to typowo dla Mozilli jest. [Chodzi o ten task](https://github.com/Varenthein/wdp-1902-03/pull/23)